Tests/Marketplace/zip_packs.py

```diff
@@ -244,12 +288,20 @@ def main():
     zipped_packs = []
     success = True
     try:
-        zipped_packs = download_packs_from_gcp(storage_bucket, gcp_path, zip_path, circle_build, branch_name)
+        download_packs_from_gcp(storage_bucket_name, gcp_path, zip_path, circle_build, branch_name)
     except Exception:
         logging.exception('Failed downloading packs')
         success = False
 
-    if remove_test_playbooks:
+    try:
+        zipped_packs = get_zipped_packs_names(zip_path, gcp_path, circle_build)
+    except Exception:
+        logging.exception('No zip files were found')
+        success = False
+
+    copy_to_other_dir(zipped_packs)
```

Comment:
```suggestion
    if private_build:
        copy_to_other_dir(zipped_packs)
```

Reply:
In the original code this copy code happened without checking for the private_build. Are you sure it's needed?

Reply:
I think it's needed to increase the readability of the code, it will help the next developer to understand it better.

Reply:
Fixed.